Makefile

```diff
@@ -166,8 +166,7 @@ unit-race:
 
 flakehunter: build
 	@$(call print, "Flake hunting integration tests.")
-	$(ITEST)
-	while [ $$? -eq 0 ]; do /bin/sh -c "$(ITEST)"; done
+	while [ $$? -eq 0 ]; do $(ITEST); done
```

Comment:
what's the rationale for not executing the itest first? i suppose this works because the call to print always succeeds?

Reply:
Exactly. Rationale was that it made no difference removing it :)

Reply:
Cool! Had never considered that, but makes sense to me :)